aten/src/ATen/core/op_registration/op_registration.h

```diff
@@ -292,11 +264,11 @@ class CAFFE2_API RegisterOperators final {
     }
 
     // TODO Remove impl_unboxedOnlyKernel once all of aten can generate boxed kernels
-    template<class FuncType, FuncType* kernel_func>
+    template<class FuncType>
     // enable_if: only enable it if FuncType is actually a function
-    guts::enable_if_t<guts::is_function_type<FuncType>::value, Options&&> impl_unboxedOnlyKernel(TensorTypeId dispatch_key) && {
```

Comment:
this one is non-public and could be changed, but it would break XLA and other extension backends. You'd have to coordinate with changes there so they don't break. Also, I don't think you get any binary size improvements from changing this.